app/models/aid_application.rb

```diff
@@ -493,6 +493,18 @@ def send_disbursement_notification
     end
   end
 
+  def activation_code_notification_method
+    if sms_consent? && email_consent?
+      "text message at #{phone_number} and email at #{email}"
+    elsif sms_consent?
+      "text message at #{phone_number}"
+    elsif email_consent?
+      "email at #{email}"
+    else
+      "no contact info"
+    end
+  end
```

Comment:
Because this output formatting, it would better live inside of the ApplicationHelper, e.g. `activation_code_notification_description(aid_application)`

Reply:
done

Reply:
Done. It seems that both in Drai and gcf, helper methods aren't tested so I deleted the associated test.